File: 5 grade calculator.js
// script.js

function calculateAverage() {
    const grade1 = parseFloat(document.getElementById('grade1').value);
    const grade2 = parseFloat(document.getElementById('grade2').value);
    const grade3 = parseFloat(document.getElementById('grade3').value);
    const grade4 = parseFloat(document.getElementById('grade4').value);

    if (isNaN(grade1) || isNaN(grade2) || isNaN(grade3) || isNaN(grade4)) {
        document.getElementById('result').innerText = 'Please enter valid numbers for all grades.';
        return;
    }

    const average = (grade1 + grade2 + grade3 + grade4) / 4;
    document.getElementById('result').innerText = `Your average grade is: ${average.toFixed(2)}`;
}

File: 5 grade calculator.py
num_subjects = int(input("Enter the number of subjects: "))
marks = []

for i in range(num_subjects):
    mark = float(input(f"Enter marks for subject {i+1}: "))
    marks.append(mark)

average = sum(marks) / len(marks)

if average >= 90:
    grade = "A+"
elif average >= 80:
    grade = "A"
elif average >= 70:
    grade = "B"
elif average >= 60:
    grade = "C"
elif average >= 50:
    grade = "D"
else:
    grade = "F"

print(f"Your final grade is: {grade}")
